app/models/ext_management_system.rb

```diff
@@ -175,18 +175,30 @@ def self.create_from_params(params, endpoints, authentications)
 
   def edit_with_params(params, endpoints, authentications)
     tap do |ems|
+      endpoints_changed       = false
+      authentications_changed = false
```

Comment:
NOTE I'm not sure how necessary it is to differentiate between endpoints and authentications changing.  Currently as far as I can tell we only use this information to restart workers and we need to do that when either endpoints or authentications change.